### PR TITLE
Sessions

### PR DESCRIPTION
**Pull request by knikolla**
- Removed Auth model object, used sessions instead
- Added caching for session objects
- Added endpoint urls to the config file
- Added generic aggregation

### CODE REVIEW

File: mixmatch/tests/unit/test_services.py [hunk 1]
```diff
@@ -19,5 +19,5 @@ class TestNothing(testcase.TestCase):
     def setUp(self):
         super(TestNothing, self).setUp()
 
-    def test_nothing(self):
+    def test_construct_url(self):
         self.assertEqual(1, 1)
```
Comment (gsilvis) [inline mixmatch/tests/unit/test_services.py]: This test... doesn't test constructing a URL

**Comment by gsilvis** [comment]
I think we can put all of the ``services`` stuff in one file---it's really not that long.

**Comment by gsilvis** [comment]
Beyond that, it looks pretty good to me

**Comment by knikolla** [comment]
@gsilvis i left them separate because at some point I want to load the right module and have polymorphism do the rest.

File: mixmatch/tests/unit/test_services.py (hunk 1 above)
Comment (knikolla) [inline mixmatch/tests/unit/test_services.py]: I know, it would be testing it if i get oslo.config to work :(

**Comment by gsilvis** [comment]
That seems like overkill when we're proxying exactly two services

**Comment by knikolla** [comment]
@gsilvis probably true. updated.

**Pull request merged**